vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -1,101 +1,70 @@
-        lowercase letters, including numbers and special characters,
-        (e.g. Conference\_News). It’s mandatory.
+**2. Click the `Add` button in the upper right corner.**
 
-![image](./img/ContentTypes3.png)
+![addContentType](./img/content_types1.png)
 
--   Select the **Save** button, then you’ll see
+**3. Configure the Content Type.**
 
-    -   **Content Type Information**:
+When adding a Content Type it is mandatory to enter `Code` and `Name` values consistent with the following parameter definitions:
 
-        -   **Code**. The value is pre-filled.
+- `Code`: A unique identifier of the Content Type that must be 3 uppercase letters (e.g. CNN).
 
-        -   **Name**. The name is pre-filled.
+- `Name`: A string value (50 characters max) consisting of uppercase and lowercase letters, numbers and/or special characters (e.g. Conference\_News).
 
-    -   **Metadata**
+Field selections in the Metadata and Attributes sections are not mandatory, but to specify a meaningful Content Type you must choose the appropriate `Type` from the Attributes drop-down (e.g. 'Text'). The available Attribute Types define Attributes to characterize the Content.
 
-        -   **Page for on-the-fly publishing**
+![configureContentType](./img/content_types2.png)
 
-        -   **Default content template**
+**4. Click the `Add` button.**
 
-        -   **Default content template for lists**
+This initializes the configuration of the selected Attribute Type. Each Attribute Type requires its own configuration. 
 
-*Default content template* and *Default content template for lists* are used
-respectively to show the detailed view of the Content and its
-representation as an element in a list.
+### Attribute Configuration
 
--   **Attributes**
+Adding an Attribute Type will load the "Add attribute" page, where `Type` is pre-filled with your selection. Follow the steps below to configure the Attribute.
 
-In the Attributes section, select the **Type** from the drop-down list.
-The list contains all the available attributes types that define which
-properties/attributes will characterize the Content.
+**1. Define the characteristics of the Attribute:**
 
-![image](./img/ContentTypes4.png)
+![configureAttributeType](./img/content_types3.png) 
 
-The Add button will start the configuration of the single attribute
-type. Every attribute type has its own configuration.
+- `Code`: It is mandatory to enter a unique name for the Attribute key (both Attributes of a 'Composite' Attribute Type may share a `Code` name). This field supports a string value (10 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. title).
 
-For example, to configure the **Text** Attribute Type, choose Text from
-the Types drop-down menu and then press the Add button.
+- `Name`: Enter a description of the Attribute. This field supports a string value (50 characters max) of uppercase and lowercase letters, numbers and/or special characters (e.g. Title).
```

Comment:
same, join parenthesis. I think it is also easier to pick out the information when it is at the end, so if anyone is scanning for the max allowed, it's easier to find at the end.